scripts/qa_crowbarsetup.sh

```diff
@@ -1715,6 +1723,10 @@ function custom_configuration()
             fi
         ;;
         database)
+            if [ -n "$want_sles12_controller" ] && iscloudver 6plus ; then
```

Comment:
Ok, cool! I merged this changes into your commit. I hope that you are fine with this